man/systemd.network.xml

```diff
@@ -816,8 +816,9 @@
       <varlistentry>
         <term><varname>IgnoreCarrierLoss=</varname></term>
         <listitem>
-          <para>A boolean. Allows networkd to retain both the static and dynamic configuration of the
-          interface even if its carrier is lost. Defaults to false.
+          <para>Takes a boolean. Allows networkd to retain both the static and dynamic configuration
+          of the interface even if its carrier is lost. When unset, the value specified with
+          <option>ConfigureWithoutCarrier=</option> is used.
```

Comment:
i thinkt the explanation on ConfigureWithoutCarrier= should have a similar hint, i.e. that it propagates into IgnoreCarrierLoss unles configured explicitly.

Reply:
ack, updated the commit.